openapi_core/contrib/django/requests.py

```diff
@@ -36,6 +36,11 @@ def create(cls, request):
         else:
             route = cls.path_regex.sub(
                 r'{\1}', request.resolver_match.route)
+            # Delete start marker and expression marker to allow concatenation.
+            if route[:1] == "^":
+                route = route[1:]
```

Comment:
I failed to trigger this case into a test environment as there are side effects in testing.

Reply:
@p1c2u Can you help me with it?

Reply:
Why not just add this to re_path:
`re_path('^test/test-regexp/$', lambda d: None)`
?

Reply:
I tried to add `^/` but it didn't work. Stupid mistake. It should work now.